### ISSUE TITLE
(Status: Alpha 2) Phoenix Linux Development - Let's bring back Cub!

### ISSUE THREAD

**Issue by dominichayesferen** [CubLinux/one#4]
Well, thought it'd be better starting a new Issue in terms of keeping the other one tidy for other ideas not related to Phoenix Linux, while this can be talk, hype, development, and so on, for my Rebirth of Phoenix Linux...
@RTbecard @dashohoxha You seemed interested in this project...

**Comment by dominichayesferen** [comment]
Starting this issue, another screenshot, this time, on real hardware:
![screenshot from 2017-07-19 19-36-04](https://user-images.githubusercontent.com/11057934/28385955-8678dcec-6cb9-11e7-97dc-ab8e751dc4e7.png)

**Comment by dominichayesferen** [comment]
A bit on how this baby will be developed:

- A release will happen every Ubuntu LTS Cycle, and then, if no major bugs are reported, will then fall dormant for the most part as they'd be nothing really to do for it...
But don't worry, unless announced, it'll never be dead. You could potentially even help add features to it, if you wanted, but first, this needs finishing, and releasing before all this is figured out... (I'm thinking Pull Requests/Feature Request Issues)
- This will probably then mean feren OS gets more of the love, but to be honest, Cub already looked awesome, and had enough love put into it, which can all be transferred from fresh...

**Comment by xmwx38** [comment]
I support you, comrade!  The second go is always the medal winner!  ;)

**Comment by dashohoxha** [comment]
@feren I liked Chromixium for being lightweight, simple and yet beautiful. Most of the time I work either in browser or in terminal, so this is all that I need, extravagant desktops do not impress me. It also works well and looks great on old computers and this is a plus. It is a pity that it was discontinued.

I am interested in seing it revived. I do have experience with shell scripts etc. but I have never worked in making a custom distribution. But maybe I can help with testing.

By the way, ferenOS looks interesting too.

**Comment by dominichayesferen** [comment]
![screenshot from 2017-07-20 10-30-08](https://user-images.githubusercontent.com/11057934/28413250-6b1b781a-6d36-11e7-90fe-9e2005d88611.png)
Top Panel Mode because why not...

**Comment by dominichayesferen** [comment]
Good News! It installs!

**Comment by dominichayesferen** [comment]
![screenshot from 2017-07-20 18-08-55](https://user-images.githubusercontent.com/11057934/28434326-d233110c-6d86-11e7-90af-9a721f04f531.png)
![20170720_160540](https://user-images.githubusercontent.com/11057934/28434389-fd67c660-6d86-11e7-9c47-22b7bf3438e2.jpg)
Fun Fact: For some reason, since this installer mode goes OpenBox, you can actually access the fully-featured OpenBox menu by right-clicking behind Ubiquity...

**Comment by Elbullazul** [comment]
@feren when can we expect an alpha/beta/rc release?

**Comment by dominichayesferen** [comment]
@Elbullazul Soon(TM)

**Comment by dominichayesferen** [comment]
The only issue I need to fix is that there's no file browser for Chromium when a 'Open File' is called... Other than that, it works fine, and also might need to fix Plank icon placements, if it's not already fixed...
Once Plank Placements are sorted, I'll start removing applications and installing more to get it closer to Cub, then will release the Alpha

**Comment by Elbullazul** [comment]
what's wrong with plank?

**Comment by dominichayesferen** [comment]
Item Order

**Comment by Elbullazul** [comment]
it's not in the right order? Plank now uses gsettings to determine object placement and order

**Comment by dominichayesferen** [comment]
Which Schema?

**Comment by dominichayesferen** [comment]
Nevermind, found it with Dconf's Search Awesomeness

**Comment by dominichayesferen** [comment]
The Alpha is here!
http://phoenixlinux.weebly.com/

**Comment by dashohoxha** [comment]
I tried it as a LiveCD in VirtualBox, but I get the box for username/password and I don't know what they are. Can it be fixed to do auto-login when tried as LiveCD? What are the current username/password?

**Comment by cinnamoner** [comment]
@feren & guys Count me in. Let the Phoenix rise from the ashes!

**Comment by dominichayesferen** [comment]
@dashohoxha If you get the Login Screen, the user name is currently still 'lubuntu' with no password...

**Comment by dominichayesferen** [comment]
...Auto-login works fine here

**Comment by dashohoxha** [comment]
Yes, username 'lubuntu' works (I didn't know this is the default one). But I get an error on login:
![screenshot from 2017-07-22 21 26 17](https://user-images.githubusercontent.com/1495805/28494118-99f6c4fa-6f25-11e7-8b0f-c699088a9588.png)
![screenshot from 2017-07-22 21 26 47](https://user-images.githubusercontent.com/1495805/28494117-99f25064-6f25-11e7-929f-6e6793f4a6c6.png)

**Comment by dominichayesferen** [comment]
What's in Show Details?

**Comment by falcon78921** [comment]
I just want to say that I continue to use Cub Linux everyday! I love Cub Linux! :)

I don't have a great amount of development experience within furthering Linux OS features. However, if there is anything I can do to help, please let me know! I have taken Cub Linux (Ubuntu 14.04) and upgraded it to Ubuntu 16.04. There were some features that broke (going from 14.04 to 16.04). However, it still works okay for me. 

Very eager to see Phoenix Linux! :)

**Comment by davidvanwijk** [comment]
I have used Cub Linux for a long time. Recently I stopped doing so because I understood development had been stopped. Now I understand there will be a 'restart' with Phoenix Linux... !
Keep up the good work!

**Comment by ghost** [comment]
Two questions, and forgive me if they have been answered elsewhere.

(1) How stable is the alpha? Am I correct to assume that as a fork of Cub Linux it's more or less as stable as that? Cub Linux has been rock solid for the most part. 

2) Is the alpha based on Ubuntu 16.04, or higher?

Actually, if you need help with the project, I'm interested. I have some programming experience (Python and Bash) and I have built my own Linux and FreeBSD distros from the ground up (though it's been a while).  

It seems like every time I find a good ChromeOS-like distro, it disappears into the ether (PapyrOS, Chromixium/Cub, others I have since forgotten), so I'd like to keep one of them going.

**Comment by dominichayesferen** [comment]
1. Some people are saying it's More Stable than an Alpha, but I'd like to iron out all the bugs first, before assuming Stable
2. Ubuntu 16.04, yeah

BTW, feren OS x32 is currently at top priority, so Phoenix is taking a little rest from Development, right now...

**Comment by daemonspudguy** [comment]
@feren So far so good

**Comment by lynxmedeiros** [comment]
@feren Greetings!
Congratulations on your return with Cub Linux, now Phoenix.
Used Cub Linux for a long time, it's an excellent distribution.
Is there somewhere to put the test results? any forum?

**Comment by dominichayesferen** [comment]
Here?

**Comment by lynxmedeiros** [comment]
Is there a forum for Phoenix Linux?

**Comment by dominichayesferen** [comment]
No

**Comment by lynxmedeiros** [comment]
Ok Feren.
I am now testing in the virtual box.
The installation was easy and completed the whole process.
The Phoenix boot is about 300 Mb, and is working fine.

**Comment by Facy5** [comment]
Hello feren ! Glad to see the future of CubLinux here !
I tested via Virtualbox, but I get some errors :

**Comment by Facy5** [comment]
Hi @feren ! Glad to see the future of CubLinux here !
I tested it on Virtualbox on Windows 7 SP1, but I get some errors, they are labeled in order of apparence  :
![capcublinux](https://user-images.githubusercontent.com/32790258/31574093-3c8688de-b0c8-11e7-837e-f9fc86963dd9.PNG)

For the 2nd message, these are screenshots of the details (unable to copy/paste ?) :

![image](https://user-images.githubusercontent.com/32790258/31574239-9da4469e-b0cb-11e7-9ced-64826f098afe.png)
![image](https://user-images.githubusercontent.com/32790258/31574243-b43c7f52-b0cb-11e7-805f-b65ff0e202c2.png)
![image](https://user-images.githubusercontent.com/32790258/31574256-e84783c8-b0cb-11e7-9d23-cde67d218fda.png)

I first believe it was because I put the OS in French (my mothertongue) but after retry with full straight installation (all in English and no 3rd party, no download during installation), I got the same errors.

I hope it could help you ;)

**Comment by dominichayesferen** [comment]
@Facy5 Thanks for that, I'll try to translate it later as my French is a bit too limited to read that right-away, and I'll get back to continuing Phoenix Development...

PS: I might've got an email from maybe a fellow friend of the original dev, so I asked them to make sure on why Cub was killed...

**Comment by dyoungwd** [comment]
Cub was killed due to private life restrictions - that was all and there was no one at the time to pick it up.

 Before it was killed the plan was to make a dedicated app store for it concentrating on google apps and opensource and free linux apps for more or less one click install

**Comment by giorosa18** [comment]
Are you planning on creating a new project repository for updates and such?

**Comment by dominichayesferen** [comment]
Maybe, not entirely sure, maybe when I get proper servers for PPAs...

**Comment by ghost** [comment]
Does it support official Chrome browser (the colored Chrome icon)?

**Comment by dominichayesferen** [comment]
Not just yet

**Comment by cinnamoner** [comment]
> Cub was killed due to private life restrictions - that was all and there was no one at the time to pick it up.

> Before it was killed the plan was to make a dedicated app store for it concentrating on google apps and opensource and free linux apps for more or less one click install

Relevant.

**Comment by Jacalz** [comment]
Wow this looks promising as a alternative to chrome os but I can not install it 😞
After creating a boot usb with Etcher the usb does not boot 😅 Every other Linux distro has worked, yeah seriously I am a big distro hopper 😂

**Comment by dominichayesferen** [comment]
Try 'isohybrid ISOHERE'...
I'll make sure it's Hybrid-ised in the next build, eventually.

**Comment by per4o** [comment]
So @feren how is development doing? I was looking for a linux distro to use as my main driver for my laptop and i found phoenix I like it a lot  since i liked Cub. On the page it says that is Alpha so then is it stable to run it as a main OS on my laptop or I should wait for it a little bit ?

**Comment by dominichayesferen** [comment]
On the pause for now, but I might just start it up again, to see what needs to be added still...

**Comment by per4o** [comment]
It would be great if you continued your work because is one of the simplistic OS's i have seen you can even post a donation button like they have on the Elementary OS page i mean everyone will be willing to throw in a couple of bucks for such a nice distro. I hope u do that and we see more of phoenix os in future.

**Comment by dominichayesferen** [comment]
Well, just moving it over to external, where feren OS is at, because of /home is running outta space, then I'll quickly rebuild it, and check it in Live, and then I'll start making adjustments again accordingly...

To-do:

- Fix feren GTK on GTK 3.20 (WIP)
- Make sure the new Maintenance Tool (kind of) works properly as intended, running on boot up
- Phoenix Linux
- feren OS January 2018 Snapshot
- Fix Themer's installing theme dialog
- Fix some more bugs

**Comment by per4o** [comment]
And don't forget to throw in a donation button on your page haha

**Comment by dominichayesferen** [comment]
...that's to be done, once there's actually a donation target. ;P
(The reason feren OS's Download button is just purely Download right now)

**Comment by per4o** [comment]
haha

**Comment by davidvanwijk** [comment]
I seem not to be able to create an usb which boots.
I tried with 1) unetbootin, 2) usbcreator-gtk and with 3) mkusb (terminal)
Can someone tell me what I do wrong?

**Comment by Jacalz** [comment]
I could not boot the usb i made with etcher either

**Comment by davidvanwijk** [comment]
@feren : you commented Try 'isohybrid ISOHERE. How? Is there a manual somewhere?

**Comment by dominichayesferen** [comment]
1. Open a Terminal
2. Type 'isohybrid ' without the quotes
3. Drag the ISO into the Terminal window
4. Press ENTER, and ignore the error
5. Try it again in Etcher

**Comment by davidvanwijk** [comment]
Thanks! 
It worked! I am submitting this with the live phoenix linux!

**Comment by davidvanwijk** [comment]
Did some testing, so far so good!
After booting however, there is one issue with the map Downloads. The message is to configure this map with OpenBox ... ?

**Comment by johnwayne4d** [comment]
i dont know programming for linux desktop well enough to truly help there, but I'm an avid user of Cub for to-site support machines for at risk networks. To support the cause, I'd be willing to host and setup a user forum for the beta stage and on, if you have the moderators to run it. There definitely needs to be a not 100% cloud-dependent, cost effective, secure, modern featured OS that works on older machines, with newbie-level installer which doesnt leave a lot of holes. what level of interest has there been?

**Comment by ghost** [comment]
> i dont know programming for linux desktop well enough to truly help there, but I'm an avid user of Cub for to-site support machines for at risk networks. To support the cause, I'd be willing to host and setup a user forum for the beta stage and on, if you have the moderators to run it. There definitely needs to be a not 100% cloud-dependent, cost effective, secure, modern featured OS that works on older machines, with newbie-level installer which doesnt leave a lot of holes. what level of interest has there been?

Relevant and interesting ... I'll be thinking about that.

**Comment by dominichayesferen** [comment]
Ok, sorry guys that I didn't get to release it in 2017, but I will try my best to release it in Early 2018, alright with that? (Just note there's Mocks coming up and then Exams, so revision will interrupt development of both OSs too)

**Comment by LanceHaverkamp** [comment]
**Complete boot failure.**

Downloaded alpha, Launched etcher.io to burn iso to usb,  Etcher warned that there is no partition table, so it's likely not a bootable image.  Boot indeed fails 100% of the time, failing back to the hard drive O/S.  Running an AMD A10 APU system, with nVidia Gforce GT750ti, & Xubuntu 16.04.3 LTS as the sole installed O/S.   

Re-downloaded Alpha from a different mirror, same failure.

**Comment by dominichayesferen** [comment]
Try the method above to the ISO before writing it:
```
    Open a Terminal

    Type 'isohybrid ' without the quotes

    Drag the ISO into the Terminal window

    Press ENTER, and ignore the error

    Try it again in Etcher
```

**Comment by 123Alpha** [comment]
I was very disappointed when I learned that no one was working on CUB Linux.  I have only played with a bit, and think it is the ideal solution for someone not wedded to Windows. It has (had?) most of the good features of Chromebooks, plus all the power of LINUX.
I found that the first thing I had to do with it was change the repository source to a standard Ubuntu one so I could install all the programs I use regularly in LINUX.
I have been able to upgrade the LINUX through 16.04, 17.04, 17.10 and now 18.04 (Alpha1). In version 16.04, you could no longer get the LINUX applications menu by right-clicking on the screen, but you could get to it through the Control Panel. After upgrading to 17, the Application menu began to work again. It still works with 18.04 Alpha1 and so far, I have had no issues with it. It is still very stable, and I have not found any glitches (which is a bit of a pleasant surprise with such an early pre-release).
If I knew how to do it, I would make a distribution from my running system!
I will try Phoenix -- hopefully it will have the latest Chromium as the Chromebook look-alike. My reason for experimenting with it is not for my own use, but to help someone who is not computer literate to be able to have the simplicity of a chromebook on an old, inexpensive computer.

**Comment by dominichayesferen** [comment]
@123Alpha Just note, that right now, the ISO is quite out of date, because it was made in Mid 2017

**Comment by lynxmedeiros** [comment]
Some prediction for the version beta?

**Comment by dominichayesferen** [comment]
Not really, working on it though

**Comment by 123Alpha** [comment]
Is anyone working on either this version, or on CUB Linux?
I should mention that CUB Linux still works very well. I have upgraded the underlying Ubuntu to version 18.04 Alpha 1 (Bionic Beaver) and although I don't recommend using such early Alpha releases, so far I have had no stability issues. I am surprised that nobody seems to have continued to support and develop this excellent OS alternative for ChromeBook/Linux lovers! So far, I have been unable to find a live CD/DVD for Neverware to give it a chance. I have tried to install it on Oracle's Virtual Machine, but without success. The USB version does not give you persistence, as far as I know.

**Comment by dominichayesferen** [comment]
Sorry it has been a while since my last development post here, but I've been busy making other things + revising for GCSEs, but I recently started working on what was in the OS so far (first Alpha) and adding to it, here's what's been done so far:
1. The ISO is now Pre-ISOHYBRID'd out of the box
2. I made some tweaks to Plank, nothing too major and nothing noticeable, just some misc. changes to make sure it won't break its config due to...
3. I've been added the Google Browser Choice Application that Cub Linux came with, as well as improving it. Firstly it now automatically switches out Chromium with Chrome on all users for their Docks, so the user no longer has to do it themselves, and I'm going to soon start working on a reverse-switch path for it too.

Alpha 2 will be coming soon.

**Comment by dominichayesferen** [comment]
4. I fixed the Blueman Incoming Files Directory Does Not Exist error too that occurs in non-LiveCD Users, to fix it, it's as simple as running this command:

`sudo apt-get install dconf-cli -y && killall blueman-applet && dconf reset -f /org/blueman/transfer/shared-path`

Then Log Out and Back in again for Bluetooth Support

**Comment by dominichayesferen** [comment]
@xmwx38 @dashohoxha @falcon78921 @davidvanwijk @andyforceno @lynxmedeiros @d4zzy @giorosa18 @zoanthropy @Jacalz @per4o @johnwayne4d @123Alpha 
Don't worry, the project is NOT dead, in fact, I think it's almost ready for Alpha 2 now, and yes, it now supports normal Chrome, and in fact, you could say it does it better than the original, when it comes to pinning it to the dock.

**Comment by davidvanwijk** [comment]
Thanks, looking forward to testing!
Keep up the good work!

**Comment by dominichayesferen** [comment]
**Hopefully SourceForge will be UP AGAIN by the time Alpha 2 is ready**

**Comment by dominichayesferen** [comment]
![screenshot from 2018-03-03 12-01-22](https://user-images.githubusercontent.com/11057934/36934070-5ab4e194-1edb-11e8-8691-c36572826b30.png)
Before Chrome Switch
![screenshot from 2018-03-03 12-03-47](https://user-images.githubusercontent.com/11057934/36934072-62cf9e3c-1edb-11e8-80eb-d2cf82f4dcdc.png)
After Chrome Switch

**Comment by dominichayesferen** [comment]
The ISO (probably) uploaded to SourceForge, just gotta wait for them to fix the thousands of 500 Page Broken Tickets on their Site Support page, before I can see if it has.

**Comment by dominichayesferen** [comment]
![screenshot_20180304_163438](https://user-images.githubusercontent.com/11057934/36947885-f00af30c-1fc9-11e8-9703-00c87f927e29.png)

**Comment by dominichayesferen** [comment]
...it's now downloadable, enjoy.

**Comment by davidvanwijk** [comment]
download link please ...

Sorry, I found it ...
Is this the correct file: PhoenixOSInstaller_v2.5.9.375_x86_x64 ?
Thanks!

Op zo 4 mrt. 2018 om 18:17 schreef The feren OS Dev <
notifications@github.com>:

**Comment by davidvanwijk** [comment]
hi,
I made the iso bootable and tested the live cd.
It did not boot with error:
"NFTS partitions /dev/block/sr0 can not be mounted, error number is 10" 

Any suggestions?

**Comment by dominichayesferen** [comment]
@davidvanwijk No, Phoenix OS is an Android Distribution, Phoenix **Linux** is what you're looking for...

**Comment by davidvanwijk** [comment]
Thanks, stupid mistake of me ...
What is the difference between 'Phoenix Linux Alpha 2 attempt 2 x64.iso' and  'Phoenix Linux Alpha 2 x64.iso ?
Thanks again!

**Comment by dominichayesferen** [comment]
Attempt 2 was me trying to get SourceForge to upload to the normal amount of download mirrors, they're both THE SAME ISO, and Attempt 2 is now the main Alpha 2 ISO, due to more mirrors.

**Comment by dominichayesferen** [comment]
.-.
So, the UK Mirrors have seemingly disappeared from the reuploaded ISO, one day later.

**Comment by shoodes** [comment]
Hey! So I've been using Phoenix Linux for a few days now and it's been working great for me. However, I'm having trouble where whenever I'm opening my laptop, and log in, the cursor disappears until I restart. Is there any updates on this?

**Comment by dominichayesferen** [comment]
:thinking: Does changing the cursor in Appearance do anything? (if possible)
(hasn't happened to me)

**Comment by shoodes** [comment]
It's almost impossible to try and do that because the cursor is invisible lol. I'll try it once I get home, will post an update.

**Comment by ItsaGlitch1** [comment]
@feren I'm trying to get phoenix linux on some pcs and i learned that you only have 64 bit images and not any 32 bit images, can you please make one in 32 bit so I am able to run it?

**Comment by dominichayesferen** [comment]
Maybe, but there are outstanding issues with a x32 release:
- Chrome Support has ended for x32 Systems
- Ubuntu is no longer giving x32 ISOs for Desktop Editions of Ubuntu in 18.04 onwards, so that might signify that they want to phase out x32.

**Comment by EvolutionMod** [comment]
are plans to ride it out until 16.04 is end of life? I'll fix and debug on my end and send pull request. The original cub died because one man was maintaining it. im on phoenix right now and i love it. thank you for the distro, It's amazing.

**Comment by ashvinnihalani** [comment]
Is it possible to create a github repo for this, seems like a good idea to consolidate bugs, allows for contributers, etc.

**Comment by feren** [comment]
brb - GCSEs.

**Comment by lynxmedeiros** [comment]
@feren. I'll download the Alpha 2 and continue with good work.

**Comment by FlexCoreOS** [comment]
How do you add the chromium app list on the dock? Do you have instructions?

**Comment by dominichayesferen** [comment]
@flexcoreOS Unfortunately Chrome and Chromium App List have been discontinued by Google, and no longer exist in the code, unless you're using Chrome OS itself. Even if you were to add it, it'd just end up opening 'chrome://apps' in a new tab.

**Comment by dominichayesferen** [comment]
Sorry I haven't been here in a little while, so I'm thinking of doing the following to Phoenix Linux, once the Feren OS Inspire Icon Set update is finished:
- PPA
- 16.04 cancelled, new one from 18.04 made as Alpha 3
- Improvements.

**Comment by FlexCoreOS** [comment]
https://firework.cloud/en/linux/

Check this out.  It has a look similar to the Cub linux app launcher and since it works like a site specific browser, it has a lower memory footprint and a user can add more than google products.

Also, do you have a link to where plank can be configured like cub / chromixium?

**Comment by dominichayesferen** [comment]
Should have a shortcut in the same place it has always been in Cub, for Plank Settings...
Sorry not much has been done with Phoenix lately, but I've now got to prepare a surprise for Feren OS first just in case Cinnamon Team do anything silly with the DE.

**Comment by dominichayesferen** [comment]
Alright, sorry that I've left everyone in the dark for so long. Here's the state of affairs so far:
I'm restarting Phoenix Linux Development soon on an Ubuntu 18.04 Base. The thing is that because of making Feren OS matured, I nearly forgot about Phoenix Linux and for that reason I'm sorry.

However, this **does not** mean Phoenix is dead in the tooth. Quite the opposite actually...
For Alpha 3, there will mainly be backend changes, including a new Package Repository made for Phoenix Linux and, of course, the updated base. Sorry this has taken so long that the original Cub Linux is now EOL, people.

**Comment by dominichayesferen** [comment]
@FlexCoreOS @lynxmedeiros @ashvinnihalani @ItsaGlitch1 @shoodes @davidvanwijk @123Alpha @dashohoxha and everyone else, sorry I left you all in the shadows for 8 months, read the message above for the current Phoenix Linux status.

**Comment by dashohoxha** [comment]
@feren I don't know about the others, but for me you don't need to be sorry. I have not been holding my breath for it. I am already using LinuxMint and I am happy with it. Maybe Chromixium was not a good idea after all.

**Comment by lynxmedeiros** [comment]
@feren Hello, thanks for the attention and continuity of _Phoenix_ and now based on _Ubuntu 18.04_ will be very good. Please advise Alpha 3 release.
Thank you.

**Comment by davidvanwijk** [comment]
Well, I am still interested! Keep up the good work.
Regards, David

Op vr 3 mei 2019 14:05 schreef Dashamir Hoxha <notifications@github.com>:

**Comment by LanceHaverkamp** [comment]
I'm seeing more & more small non-profits trying to get everyone in a SAAS environment, mostly in Google.  So, I'm very interested!  Getting employees & volunteers 100% SAAS means you can stop paying for backups, anti-virus, etc.  

- Neverware's CloudReady is not a good option, it's too expensive.  
- Chromebooks are too small for older eyes & hands.
- Desktop users don't want to give-up options by committing to a chromebox.
- Windows can access G Suite, but it's hard to train everyone to not save documents to the internal hard drive.  

As I said, I'm very interested.  And I'd recommend an administrative option to disable users from saving locally; that way Phoenix/Cub could be as safe as a chromebook for preventing document loss.

**Comment by MrGKanev** [comment]
Is the project still reviving? The last comment is nearly a year ago and the domain is dead. 
As @LanceHaverkamp said Neverware's CloudReady isn't a good option not only because of the price. It isn't really compatible with a lot of hardware. It would be a great project and I would love to help as much as I can.

**Comment by dominichayesferen** [comment]
Kinda... been too busy with Feren OS unexpectedly to really touch Phoenix Linux all that much, sadly.

**Comment by dominichayesferen** [comment]
Ok, I'll be blunt here. I've been working on cancelling this, but... unfortunately I've kinda been too busy to CANCEL it the way I'd like to, as you can tell by the 2 years of silence.

Why am I taking so long to cancel it? Because I want to make one last build of it to allow people to see the final result of the aborted work, as well as get infrastructure set up so that someday someone else could potentially pick up the project from where I left off. Here's a screenshot from experimenting with a new look for this final build (planned to be based on 22.04), using a Lubuntu 22.04 VM:
![Screenshot_20221127_214653](https://user-images.githubusercontent.com/11057934/204161406-922db514-f6c7-4a96-8a14-1e8f7c467d80.png)

I'm sorry to everyone who was hoping this would happen, but I ended up lacking any time to work on Phoenix Linux nowadays to do literally anything, really, to it.
Watch this space, however, for the eventual day the final build releases.

**Comment by dominichayesferen** [comment]
![Screenshot_20221127_231504](https://user-images.githubusercontent.com/11057934/204164914-7f1a953e-fa5c-4bc4-8673-a65ae96ff2df.png)

more progress with messing with Openbox to form a potential basis for the new look, I guess.

**Comment by TheGitGubGuy** [comment]
I need someone to bring this back. I LOVE material 3/U and i wish it could be implemented with Linux like this. Just sad to see this project fail.

**Comment by dominichayesferen** [comment]
I will be sure to provide the necessary scripting to allow future carnations of Phoenix Linux to exist more easily.

Heck, that and of course focusing on finalising Feren OS 2023.1? before working on the final build, has been what is holding back the cancellation process. 😅

**Comment by TheGitGubGuy** [comment]
You better catch up because it feels like Google is updating material 3/U every month on chrome os.
